lib/modules/manager/gomod/extract.ts

```diff
@@ -32,6 +32,7 @@ function getDep(
   if (digestMatch?.groups?.digest) {
     dep.currentDigest = digestMatch.groups.digest;
     dep.digestOneAndOnly = true;
+    dep.versioning = 'loose';
```

Comment:
we probably need a go versioning?

Reply:
hopefully it has no bad side effects 🤔

Reply:
I think it should be fine here. It could alternatively be semver with ignoreUnstable=true but I don't see any advantage